Better Symply

Zwuiix-cmd opened the pull request:
- Added TextPacket check.
- Added JWT Chains check.
- Added SessionPingUpdateEvent.
- Internal Server Error reimplementations.

Commit:
Delete composer.lock

AzaleeX reviewed (approved):
💖Thank you for your contribution

Zwuiix-cmd committed:
Better Symply

- Added TextPacket check.
- Added JWT Chains check.
- Added SessionPingUpdateEvent.
- Internal Server Error reimplementations.
